src/bufr/BufrParser/Query/DataProvider/DataProvider.cpp

```diff
@@ -58,6 +63,24 @@ namespace bufr {
         }
 
         deleteData();
+
+        if (!foundBufrMsg)
+        {
+            std::ostringstream errStr;
+            errStr << "No BUFR messages were found! ";
+            errStr << "Please make sure that " << filePath_ << " exists and is a valid BUFR file.";
+            throw eckit::BadValue(errStr.str());
```

Comment:
Do these need wrapped in an IFDEF like `#ifdef BUILD_IODA_BINDING`? Or is eckit always assumed to be here when this file is compiled?

Reply:
@rmclaren just double checking, does this need an ifdef or will eckit always be available?